src/deploy/invokeAutomationTests.js

```diff
@@ -0,0 +1,25 @@
+const fetch = require('node-fetch');
+
+function invokeAutomationTests({ jenkinsUrl, devicePlatforms, whakapapaBranchName, skylineBranchName, skylineS3BranchDirName, buildJobUrl, sourceRepo }) {
+    const url = jenkinsUrl || "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
+
+    return fetch(url, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "token": "TAR-HTML5_UAT",
+        },
+        body: JSON.stringify({
+            "devicePlatforms": devicePlatforms || "Unknown,Tizen,LG,XboxOne,FireTV,AndroidTV,Vizio,Comcast",
+            "whakapapaBranchName": whakapapaBranchName,
+            "skylineBranchName": skylineBranchName,
+            "skylineS3BranchDirName": skylineS3BranchDirName,
```

Comment:
we can default this if missing, to be computed from the skylineBranchName

Reply:
This looked more appropriate to be based off the pull request rather than skyline branch.  Since this is for the test branches no?  I'll move the guard here instead of from Skyline